gitpod-io/website
# :wrench: Fix Changelog CTA from `/support`

## Description

Changelog CTA tile from [`/support`](https://deploy-preview-3575--gitpod-kumquat.netlify.app/support) page

## Related Issue(s)



Fixes #3572

## How to test



Go to [`/support`](https://deploy-preview-3575--gitpod-kumquat.netlify.app/support) & see :eyes:
